tests/internal/unit/CMakeLists.txt

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+# other BLT Project Developers. See the top-level LICENSE file for details
+# 
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+#------------------------------------------------------------------------------
+# BLT Internal Testing Project
+#------------------------------------------------------------------------------
+
+cmake_minimum_required(VERSION 3.8)
+
+project(blt-internal-tests LANGUAGES C CXX)
```

Comment:
Is it weird to have two `project` commands with the same name within a given build tree?

Reply:
copy/paste error... ill fix it. thanks for the catch!